src/redux/books/books.js

```diff
@@ -6,11 +6,19 @@ const REMOVE_BOOK = 'REMOVE_BOOK';
 const books = (state = [], action) => {
   switch (action.type) {
     case ADD_BOOK:
-      return action.book;
-
+      return [
+        ...state,
+        {
+          added: true,
+        },
```

Comment:
- [ ] In case when add book the function returns a new state object in which the books array will contain a new book object, passed by action.payload.

Reply:
@omarsalem7 Thanks for the review. Please, can you give more clarifications, I really don't want to guess what I am supposed to do here?